csrc/preseg_passes/move_split_cat.cpp

```diff
@@ -198,30 +279,102 @@ TensorView* exprsFormSplit(
   return split_in;
 }
 
-TensorView* CancelSplitCat::findCancelingSplit(
-    CatOp* cat,
-    std::vector<Expr*>& use_def_chain) {
+int64_t CancelSplitCat::computeCatAxisAfterZipping(
+    const std::vector<IterDomain*>& slice_rfactor,
+    IterDomain* cat_id) {
+  ValGraph& exact_graph = id_model_.idGraph(IdMappingMode::EXACT);
```

Comment:
```suggestion
  const ValGraph& exact_graph = id_model_.idGraph(IdMappingMode::EXACT);
```
Nitpick on this one. since we mentioned earlier that `id_model_` is not supposed to be updated.
`toGroup` and `getDefinitions` are both const methods.